### ISSUE TITLE
async_setup_platforms instead of awaiting async_forward_entry_setups; this will fail in version 2023.3.

### ISSUE THREAD

**Issue by canedje** [Sholofly/lghorizon#43]
**Describe the bug**
I do get a warning from HA that something will be depreciated in version 2023.3

```
Logger: homeassistant.helpers.frame
Source: helpers/frame.py:77
First occurred: 07:25:40 (1 occurrences)
Last logged: 07:25:40

Detected integration that called async_setup_platforms instead of awaiting async_forward_entry_setups; this will fail in version 2023.3. Please report issue to the custom integration author for lghorizon using this method at custom_components/lghorizon/__init__.py, line 56: hass.config_entries.async_setup_platforms(entry, PLATFORMS)

**Comment by pjannink** [comment]
Same here. Running HA 2023.2 in Docker on Synology NAS

 Logger: homeassistant.helpers.frame
Source: helpers/frame.py:77
First occurred: 16:54:05 (1 occurrences)
Last logged: 16:54:05
Detected integration that called async_setup_platforms instead of awaiting async_forward_entry_setups; this will fail in version 2023.3. Please report issue to the custom integration author for lghorizon using this method at custom_components/lghorizon/__init__.py, line 56: hass.config_entries.async_setup_platforms(entry, PLATFORMS)

**Comment by dynasticorpheus** [comment]
#41 Should fix this

**Comment by Sholofly** [comment]
Resolved by #41
